tests/Unit/PointwiseFunctions/SpecialRelativity/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY "Test_SpecialRelativity")
+
+set(LIBRARY_SOURCES
+  Test_LorentzBoostMatrix.cpp
+  )
+
+add_test_library(
```

Comment:
Note that we have *not* updated the test CMake files yet, so leave the test CMake file as-is.